sdk/textanalytics/azure-ai-textanalytics/samples/async_samples/sample_analyze_action_healthcare_async.py

```diff
@@ -0,0 +1,126 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+"""
+FILE: sample_analyze_action_healthcare_async.py
+
+DESCRIPTION:
+    This sample demonstrates how to submit a collection of text documents for analysis, which uses the
+    AnalyzeHealthcareEntitiesAction and RecognizePiiEntitiesAction to recognize healthcare entities,
+    along with any PII entities.
+    The response will contain results from each of the individual actions specified in the request.
+
+USAGE:
+    python sample_analyze_action_healthcare_async.py
+
+    Set the environment variables with your own values before running the sample:
+    1) AZURE_LANGUAGE_ENDPOINT - the endpoint to your Language resource.
+    2) AZURE_LANGUAGE_KEY - your Language subscription key
+"""
+
+
+import os
+import asyncio
+
+
+async def sample_analyze_action_healthcare():
+    from azure.core.credentials import AzureKeyCredential
+    from azure.ai.textanalytics.aio import TextAnalyticsClient
+    from azure.ai.textanalytics import (
+        AnalyzeHealthcareEntitiesAction,
+        RecognizePiiEntitiesAction,
+    )
+
+    endpoint = os.environ["AZURE_LANGUAGE_ENDPOINT"]
+    key = os.environ["AZURE_LANGUAGE_KEY"]
+
+    text_analytics_client = TextAnalyticsClient(
+        endpoint=endpoint,
+        credential=AzureKeyCredential(key),
+    )
+
+    documents = [
+        """
+        Patient needs to take 100 mg of ibuprofen, and 3 mg of potassium. Also needs to take
+        10 mg of Zocor.
+        """,
+        """
+        Patient needs to take 50 mg of ibuprofen, and 2 mg of Coumadin.
+        """
+    ]
+
+    async with text_analytics_client:
+        poller = await text_analytics_client.begin_analyze_actions(
+            documents,
+            display_name="Sample Text Analysis",
+            actions=[
+                AnalyzeHealthcareEntitiesAction(),
+                RecognizePiiEntitiesAction(),
```

Comment:
since this is a healthcare flavor of sample, maybe we pass the `domain_filter="phi"` option here?

Reply:
Done